charts/plantuml/README.md

```diff
@@ -71,6 +71,7 @@ helm upgrade --install plantuml stevehipwell/plantuml --version 3.36.0
 | ingress.enabled | bool | `false` | If `true`, create an `Ingress` resource. |
 | ingress.hosts | list | See _values.yaml_ | Ingress hosts. |
 | ingress.ingressClassName | string | `nil` | Ingress class name. |
+| ingress.path | string | / | Ingress path. |
```

Comment:
Did you run `helm-docs` as I'd expect to see `"/"`.

Reply:
Now I did :-)